src/Core/Session/Factory/Session.php

```diff
@@ -82,10 +80,15 @@ public function createSession(App\Mode $mode, App\BaseURL $baseURL, IManageConfi
 							$handler = new Handler\Cache($cache, $logger);
 						}
 						break;
+					default:
+						$handler = null;
 				}
 
 				$session = new Type\Native($baseURL, $handler);
 			}
+		} catch (\Throwable $e) {
+			$logger->notice('Unable to create session', ['mode' => $mode, 'session_handler' => $session_handler, 'exception' => $e]);
```

Comment:
```suggestion
			$logger->alert('Unable to create session', ['mode' => $mode, 'session_handler' => $session_handler, 'exception' => $e]);
```

Reply:
I'm not using a high log level because they might be reasons why a session can't be instantiated, like in backend mode or in CLI mode. In these mode we still offer a degraded script-only session so the log shouldn't fill with high-priority messages for an expected behavior.